backend/tests/prisonerLocationHistory.test.js

```diff
@@ -28,7 +30,26 @@ describe('Prisoner location sharing history', () => {
     prisonApi.userCaseLoads = jest.fn().mockResolvedValue([])
     prisonApi.getPrisonerDetail = jest.fn()
 
-    controller = prisonerLocationHistory({ prisonApi, logError })
+    prisonApi.getStaffDetails = jest.fn().mockResolvedValue({ firstName: 'Joe', lastName: 'Bloggs' })
+    caseNotesApi.getCaseNote = jest.fn().mockResolvedValue({ text: 'Some details regarding what happened' })
+
+    const caseNotesTypes = [
```

Comment:
These are duplicated further down. If you pull them out of the describe you can use them everywhere in the module